openapi/paths/subscriptions@{id}@upcoming-invoice.yaml

```
@@ -0,0 +1,31 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Orders
+  summary: Retrieve subscription order's upcoming invoice
+  operationId: GetSubscriptionUpcomingInvoice
+  x-sdk-operation-name: getUpcomingInvoice
+  description: |-
+    Retrieve an upcoming invoice from the specified subscription order.
+
+    The endpoint is temporary before upcoming invoices get a complete
+    integration.
+  parameters:
+    - $ref: ../components/parameters/collectionExpand.yaml
+  deprecated: true
+  responses:
+    '200':
+      description: Upcoming invoice is retrieved.
```

nitpick: the word 'is' is not needed here
```suggestion
      description: Upcoming invoice retrieved.
```